test/test_pep518.py

```diff
@@ -47,3 +49,8 @@ def test_pep518(tmp_path):
     # check that the expected wheels are produced
     expected_wheels = utils.expected_wheels("spam", "0.1.0")
     assert set(actual_wheels) == set(expected_wheels)
+
+    assert not (project_dir / "42").exists()
```

Comment:
Might be worth adding a comment why these asserts are here. Feel free to link to #421 if it's easier